tide_media.install

```diff
@@ -150,3 +150,79 @@ function tide_media_update_8002() {
     }
   }
 }
+
+/**
+ * Video transcript field change from text field to WYSIWYG
+ */
+function tide_media_update_8003() {
+  $database = Drupal::database();
+  $line_breaks = [
+    '\'\r\n\'',
+    '\'\n\r\'',
+    '\'\n\'',
+    '\'\r\'',
+  ];
+  //step1 create two temporary tables for holding data
```

Comment:
Inline comments must start with a single space, the first letter must be capitalised, and the sentence must end with a period `.`.